src/types/types.ts

```diff
@@ -136,6 +136,73 @@ export interface Token {
     tokenStandard: TokenStandard;
 }
 
+export interface TokenMetadataResponse {
```

Comment:
This doesn't look like it points to our new metadata endpoint with `onChainAccountInfo` and `onChainMetadata`
I also see no usage of this type, do you plan on adding a method for the sdk for it?
https://docs.helius.xyz/solana-apis/token-metadata-api#v0-token-metadata

Reply:
I realized I used the wrong example to create this type. I have now updated with the correct properties including onChainAccountInfo and legacyMetadata. Thank you.